etna/lib/commands.rb

```diff
@@ -696,4 +696,36 @@ def setup(config)
       super
     end
   end
+
+  class RunJob < Etna::Command
+    include WithEtnaClients
+
+    def job_map
+      {
+        "file_discovery" => SftpFileDiscoveryJob
+      }
+    end
```

Comment:
rather than maintaining this map you might do this with some metaprogramming, like job = Kernel.const_get("#{job_name}_job".camelize.to_sym)